snappi_ixnetwork/snappi_api.py

```diff
@@ -611,6 +635,9 @@ def _validate_instance(self, config):
                 )
                 self.add_error(msg)
                 self.warning(msg)
+        self._initial_flows_config = self.config().flows.deserialize(
+            config.flows.serialize(config.flows.DICT)
+        )
```

Comment:
```serialize``` is a costly operation. It will better if we serialize at the time of update. We already store current config in ```self._running_config```.  So we can utilize it.

Reply:
`self._running_config` is a reference to the config. Since we are updating the same flow object, can't compare the flows to be updated with the existing flows.

We need initial configuration to compare with the existing configuration. Can't think of other thing than serialize, please let me know if there is better approach to handle this.